sdk/identity/Azure.Identity/README.md

```diff
@@ -22,85 +22,62 @@ Install-Package Azure.Identity
 
 ### Prerequisites
 * An [Azure subscription][azure_sub].
-* An existing Azure Active Directory service principal. If you need to create a service principal, you can use the Azure Portal or [Azure CLI][azure_cli].
+* The [Azure CLI][azure_cli] can also be useful for authenticating in a development environment, creating accounts, and managing account roles.
 
-#### Creating a Service Principal with the Azure CLI
-Use the [Azure CLI][azure_cli] snippet below to create/get client secret credentials.
+### Authenticating the development environment
 
- * Create a service principal and configure its access to Azure resources:
-    ```bash
-    az ad sp create-for-rbac -n <your-application-name> --skip-assignment
-    ```
-    Output:
-    ```json
-    {
-        "appId": "generated-app-ID",
-        "displayName": "dummy-app-name",
-        "name": "http://dummy-app-name",
-        "password": "random-password",
-        "tenant": "tenant-ID"
-    }
-    ```
+When debugging and executing code locally it is typical for a developer to use their own account for authenticating calls to Azure services. There are several developer tools which can be used to perform this authentication in your development environment.
 
-#### Authenticate the client
+#### Authenticating via Visual Studio 
```

Comment:
Analyzer has failed, suggesting that we should still have `Authenticate the client` header in `Getting started` section.